File: app/src/main/java/com/dev/games/sudoku/SplashActivity.java
package com.dev.games.sudoku;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.dev.games.sudoku.sudoku_game.SudokuArray;

public class SplashActivity extends AppCompatActivity {

    private SudokuArray mSudokuArray;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mSudokuArray = SudokuArray.getInstance(this);
        Intent intent = new Intent(this, MenuActivity.class);
        startActivity(intent);
        finish();
    }

}

File: app/src/main/java/com/dev/games/sudoku/SplashActivity.kt
package com.dev.games.sudoku

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.dev.games.sudoku.sudoku_game.SudokuArray

class SplashActivity : AppCompatActivity() {

    private var mSudokuArray: SudokuArray? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mSudokuArray = SudokuArray.getInstance(this)
        val intent = Intent(this, MenuActivity::class.java)
        startActivity(intent)
        finish()
    }

}
